src/lib/statetable.rs

```diff
@@ -1,14 +1,18 @@
 use std::collections::hash_map::{Entry, HashMap};
 
-use grammar::{PIdx, Grammar, Symbol};
+use grammar::{AssocKind, Grammar, Precedence, PIdx, Symbol};
 use stategraph::StateGraph;
 
 /// A representation of a StateTable for a grammar. `actions` and `gotos` are split into two
 /// separate hashmaps, rather than a single table, due to the different types of their values.
 #[derive(Debug)]
 pub struct StateTable {
-    pub actions: HashMap<(usize, Symbol), Action>,
-    pub gotos  : HashMap<(usize, PIdx), usize>
+    pub actions      : HashMap<(usize, Symbol), Action>,
```

Comment:
Yes, but why?

On Friday, 15 July 2016, Laurence Tratt <notifications@github.com> wrote:

> In src/lib/statetable.rs
> <https://github.com/softdevteam/lrtable/pull/37#discussion_r70982103>:
>
> >  use stategraph::StateGraph;
> >
> >  /// A representation of a StateTable for a grammar. `actions` and `gotos` are split into two
> >  /// separate hashmaps, rather than a single table, due to the different types of their values.
> >  #[derive(Debug)]
> >  pub struct StateTable {
> > -    pub actions: HashMap<(usize, Symbol), Action>,
> > -    pub gotos  : HashMap<(usize, PIdx), usize>
> > +    pub actions      : HashMap<(usize, Symbol), Action>,
>
> Ditto.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/softdevteam/lrtable/pull/37/files/0f7fb8c6ca478869042c7d04d4eea929cb7d9a73#r70982103>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAIKZGwl-4Z5xDgzWlh-YI5EB38xyCRbks5qV5lSgaJpZM4Hlhr5>
> .
>